### PR TITLE
Add additional info on color, break up cvar tables

### PR DESCRIPTION
**Pull request by storytellermahkasad**
### Summary
Split cvar tables to reduce visual noise. Updates documentation on the -color paran of tembed and embed

#### PR Type

- [ ] This PR is a code change that implements a feature request.
- [ ] This PR fixes an issue.
- [x] This PR adds a new feature that is not an open feature request.
- [ ] This PR is not a code change (e.g. documentation, README, ...)
#### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] If code changes were made then they have been tested.
- [X] I have updated the documentation to reflect the changes.

### CODE REVIEW

File: docs/aliasing/api.rst [hunk 1]
```diff
@@ -170,41 +170,53 @@ second \"third word\" words
 >>> !asdf first "second arg"
 ['first', 'second arg']
 
-.. _cvar-table:
+.. _attribute-cvar-tables:
 
-Cvar Table
-----------
-This table lists the available cvars when a character is active.
+Attribute Cvar Tables
```
Comment (storytellermahkasad) [inline docs/aliasing/api.rst:175]: Will change to Table with other decisions.

File: docs/aliasing/api.rst [hunk 2]
```diff
@@ -170,41 +170,53 @@ second \"third word\" words
...
+
+Other Cvars Tables
+------------------
+This table lists the other available cvars when a character is active.
+
+================ =========================================== ====
+Name             Description                                 Type
+================ =========================================== ====
+armor            Armor Class.                                int
+color            The CSettings color for the character       hex
```
Comment (storytellermahkasad) [inline docs/aliasing/api.rst:212]: Uncertain what the Type should be here.

Comment (storytellermahkasad) [reply docs/aliasing/api.rst]: Handled as string as that's the internal type

File: cogs5e/pbpUtils.py [hunk 3]
```diff
@@ -58,9 +59,10 @@ async def embed(self, ctx, *, args):
         embed.set_image(url=args.last('image', '') if 'http' in str(args.last('image')) else '')
         embed.set_footer(text=args.last('footer', ''))
         try:
-            embed.colour = int(args.last('color', "0").strip('#'), base=16)
-        except:
-            pass
+            color = args.last('color')
+            embed.colour = int(color.strip('#'), base=16)
+        except (AttributeError, ValueError):
+            embed.colour = randint(0, 0xffffff)
```
Comment (zhudotexe) [inline cogs5e/pbpUtils.py:65]:
```suggestion
            pass
```
The color is already initalized to random in the constructor, this just keeps the import cleaner, and if we ever decide to change the default color in the future for any reason, it's one less refactor

**Review by zhudotexe — APPROVED** [review]

**Pull request merged**